packages/core/src/html/CustomListIconProcessor.ts

```diff
@@ -25,9 +25,40 @@ interface IconAttributes {
   once?: boolean;
 }
 
+class TextsManager {
+  texts: string[] = [];
+  pt: number = 0;
+  size: number = 0;
```

Comment:
Changed;
Though personally I prefer maintaining the size variable (as c++ STL style). 
But you are correct in pointing out that this is unnecessary.
It could be a source of bug if future developer fails to handle (simple code like this) properly. So I remove it.